api/api.proto

```diff
@@ -1585,3 +1585,81 @@ message SdkCloudBackupStateChangeRequest {
 
 message SdkCloudBackupStateChangeResponse {
 }
+
+enum CloudMigrateOperationType {
+    // Migrate all volumes in the cluster
+    MigrateCluster = 0;
+    // Migrate a single volume
+    MigrateVolume = 1;
+    // Migrate a group of volumes
+    MigrateVolumeGroup = 2;
+}
+
+enum CloudMigrateStage {
+    Backup = 0;
```

Comment:
All enums must be unique across the entire file (according to protoc error messages). Please prefix with the enum name.

Reply:
Scoped them inside a message which is easier to read then prefixing with the enum name